salt/tests/unit/modules/files/test_metalk8s_checks.yaml

```
@@ -601,3 +609,44 @@ route_exists:
     error: >-
       A route was found for 10.96.0.0, but it does not match the full
       destination network 10.96.0.0/12
+
+containerd_filesystem:
+  # 1. Success Not XFS
+  - run_all_df_ret:
+      stdout: "Type\next4"
```

nitpick: Maybe you can put the full string not using `\n` so that we easily see that it's a df output

```suggestion
      stdout: |-
        Type
        ext4
```